### PR TITLE
flutter version upgrade - sample interstitial

### PR DESCRIPTION
**Pull request by bhargavraviya**
## Description

- upgrade the new flutter version according to 3.13.9 from scratch
- pubspec.yaml remove line
- remove old icons and add new icons of Flutter
- also native Android and iOS changes

### CODE REVIEW

File: samples/admob/interstitial_example/android/app/build.gradle [hunk 1]
```diff
@@ -46,12 +44,12 @@ android {
...
         applicationId "com.example.interstitial_example"
         // You can update the following values to match your application needs.
-        // For more information, see: https://docs.flutter.dev/deployment/android#reviewing-the-build-configuration.
+        // For more information, see: https://docs.flutter.dev/deployment/android#reviewing-the-gradle-build-configuration.
         minSdkVersion 19
-        multiDexEnabled true
         targetSdkVersion flutter.targetSdkVersion
         versionCode flutterVersionCode.toInteger()
         versionName flutterVersionName
+        multiDexEnabled true
```
Comment (malandr2) [inline samples/admob/interstitial_example/android/app/build.gradle:52]: Nit: keep order alphabetiecal

**Review by malandr2 — APPROVED** [review]

**Comment by malandr2** [comment]
Can you apply these changes to the rest of the samples? Thanks!

**Comment by bhargavraviya** [comment]
> Can you apply these changes to the rest of the samples? Thanks!

Yes 
I have changes in the rest of all samples. Give me some time for that.

**Comment by bhargavraviya** [comment]
> Can you apply these changes to the rest of the samples? Thanks!

@malandr2 

Samples Of Google Mobile Ads

- [x ]  Adaptive Banner Example #962
- [ ]  App Open Example #965
- [ ]  Mediation Example
- [ ]  Native Platform Example
- [ ]  Native Template Example
- [x ]  Rewarded Example #963 
- [x ]  Rewarded Interstitial Example #964

Some points are pending

**Pull request merged**